dali/operators/reader/numpy_reader_op.cc

```diff
@@ -98,6 +106,48 @@ sizes vary a lot.)code", true)
       R"code(If set to True, the header information for each file is cached, improving access
 speed.)code",
       false)
+    .AddOptionalArg<std::vector<int>>("roi_start",
```

Comment:
I would add a note that these options are not supported for the GPU variant.

Reply:
done